autoPyTorch/constants_forecasting.py

```diff
@@ -0,0 +1,19 @@
+# The cosntant values for time series forecasting comes from
```

Comment:
can we not have the contents of this file in [`autoPyTorch/constants.py`](https://github.com/dengdifan/Auto-PyTorch/blob/4898ca5ef492b465ad1331df2b380b10be070087/autoPyTorch/constants.py)